audio/mods/mod_xm_s3m.cpp

```diff
@@ -63,20 +63,18 @@
 
 #include "common/debug.h"
 #include "common/file.h"
-#include "common/textconsole.h"
 #include "common/memstream.h"
 
 #include "audio/audiostream.h"
-#include "audio/decoders/wave.h"
 #include "audio/mods/mod_xm_s3m.h"
 #include "audio/mods/module2wav.h"
 
-#define DEBUG_DUMP_FILE 0
-
 namespace Modules {
 
-class ModXmS3mConvertor {
+class ModXmS3mStream : public Audio::AudioStream {
 private:
+	static const int NUM_MIX_BUFFER = 16384;
```

Comment:
Are you sure about the mix buffer size? The original computes it dynamically in `calculate_mix_buf_len`.

Reply:
You're right, it's better to use calculate_mix_buf_len for it.